modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -1439,7 +1439,7 @@ protected void waitForDiscovery(Ignite... ignites) throws IgniteCheckedException
      */
     protected IgniteEx grid(String name) {
         if (!isRemoteJvm(name))
-            return (IgniteEx)G.ignite(name);
+            return (IgniteEx)IgnitionEx.grid(name);
```

Comment:
Why do we need it? I think it can fail only if we cast to `IgniteEx` inside test and can affect only sandbox tests, all other tests should work well. If we cast to `IgniteEx` inside sandbox test we should better rewrite this logic.